assets/locales/service.cy.toml

```diff
@@ -12,3 +12,29 @@
 # count - 4 = other; 4,5 & >6
 # Plural rules can be found here: http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html
 #-------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+[Variable]
+description = "Variable"
+one = "Variable"
+other = "Variables"
+
+[VariableExplanation]
+description = "What the variable means"
+one = "What does the variable mean"
+other = "What do the variables mean"
```

Comment:
The wording around this explanation is being changed in `dp-frontend-dataset-controller` so it might be worth checking if the same should apply here.

[https://github.com/ONSdigital/dp-frontend-dataset-controller/pull/266/files](https://github.com/ONSdigital/dp-frontend-dataset-controller/pull/266/files)

Reply:
Will update the copy. Just noted that the question mark is coming from the `dp-renderer` so will remove it and get another PR for that service. You'll need to update the reference to the renderer in the frontend dataset controller to complete your task too ^^

Reply:
https://github.com/ONSdigital/dp-renderer/pull/66